CheckDrive.Api/CheckDrive.Infrastructure/Persistence/Configurations/RoleConfigurations.cs

```diff
@@ -0,0 +1,57 @@
+﻿using CheckDrive.Application.Constants;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace CheckDrive.Infrastructure.Persistence.Configurations;
+
+class RoleConfigurations : IEntityTypeConfiguration<IdentityRole>
+{
+    public void Configure(EntityTypeBuilder<IdentityRole> builder)
```

Comment:
this should be inside of DbContext, since other configuration is there for Identity. We should keep all of them in one place. With separate configuration we would have half in one place, another half in different place.